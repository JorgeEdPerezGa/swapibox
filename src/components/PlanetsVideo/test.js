import React from 'react';
import PlanetVideo from './index';
import setupTest from '../../setupTests';
import { shallow } from 'enzyme';

describe('<PlanetVideo/>', () => {
  const wrapper = shallow(<PlanetVideo />);

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
