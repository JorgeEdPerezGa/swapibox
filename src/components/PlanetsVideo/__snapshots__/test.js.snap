// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlanetVideo/> should match snapshot 1`] = `<PlanetsVideo />`;
